roles/Cargo.lock

```diff
@@ -1,12 +1,12 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
+version = 4
```

Comment:
this breaks MSRV

Reply:
I highly recommend locking SRI development setup to Rust 1.75.0

that's what I did with my setup when the same thing happened to me

Reply:
@jbesraa you missed this.